### PR TITLE
Us20 borrower views deed

### PR DESCRIPTION
**Pull request by Kirosoft**
Add the /deed/<id> GET entry point

### CODE REVIEW

File: application/deed/views.py [hunk 1]
```diff
@@ -15,6 +14,18 @@
 _title_validator = None
 
 
+@deed_bp.route('/<deed_reference>')
```
Comment (danielmarker) [inline application/deed/views.py]: Will be safer to add a "Get" method here to avoid confusion

**Comment by danielmarker** [comment]
One Comment left.

**Comment by danielmarker** [comment]
Code is good, happy with this :+1:

File: application/deed/views.py (hunk 1 above)
Comment (tsangsimon) [inline application/deed/views.py]: Yep, definitely good practice to be explicit on the function

**Comment by danielmarker** [comment]
Happy with the correction after Simon's Comment :+1:

**Pull request merged**